errai-data-binding/pom.xml

```diff
@@ -93,17 +93,33 @@
       <classifier>sources</classifier>
       <scope>provided</scope>
     </dependency>
-    
+
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.jboss.errai</groupId>
+      <artifactId>errai-data-binding-export-apt</artifactId>
+      <scope>compile</scope>
```

Comment:
If you put this as compile scope, it will be transitively included in the classpath of webapps using data-binding. Is that intended or did you want to use provided?

Reply:
It's intended. Every module, inside errai or not, that has errai-data-binding as a dependency should get errai-data-binding-export-apt as well, otherwise, its @Bindable types wouldn't be exported in an Export File and therefore not be visible to an Errai App.